server/s3_object_metadata.cc

```diff
@@ -605,8 +605,10 @@ std::string S3ObjectMetadata::to_json() {
   root["create_timestamp"] = current_time.get_isoformat_string();
 
   Json::FastWriter fastWriter;
-  return fastWriter.write(root);
-  ;
+  std::string s = fastWriter.write(root);
+  s3_log(S3_LOG_INFO, request_id,
```

Comment:
We are trying to reduce logging as much as possible, due to multiple s3servers... better if INFO mode logging is changes to DEBUG mode

Reply:
Removed completely. It was used to understand what kind of metadata is stored for each object, and it's no longer needed.